egs/swbd/s5c/local/chain/tuning/run_tdnn_blstm_1b.sh

```diff
@@ -0,0 +1,254 @@
+#!/bin/bash
+
+# tdnn_blstm_1b is same as tdnn_blstm_1b, but with the more tdnn layers
```

Comment:
**SWBD**
Yes I meant 1a. There number of parameters increases in tdnn_lstm_1a and tdnn_lstm_1b due to the TDNN layers.

**AMI**
For the ami recipe where I just changed the lstmp layers to fast-lstmp layers there is no increase in number of parameters.